src/mon/PGMonitor.cc

```diff
@@ -190,6 +190,8 @@ void PGMonitor::update_from_paxos(bool *need_bootstrap)
     int r = get_version(pg_map.version + 1, bl);
     if (r == -ENOENT) {
       dout(10) << __func__ << " failed to read_incremental, read_full" << dendl;
+      // reset pg map
+      pg_map = PGMap();
```

Comment:
@jecluis if the incremental / full version pgmap is missing, we will have assert failure. i am not sure that's the best way to address this though.